dask/bytes/s3.py

```diff
@@ -13,15 +14,46 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_s3(storage_options=None):
```

Comment:
I would normally agree except when arguments are actually used within the function.  I generally leave arguments in kwargs if they just get passed through but expand them explicitly if they're actively used within the function.  It makes managing them easier (no dict management within ordinary code) and it informs other developers to read the function what active variables they need to be aware of.